runtime/server/chat.go

```diff
@@ -129,10 +130,20 @@ func (s *Server) Complete(ctx context.Context, req *runtimev1.CompleteRequest) (
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
+	analystAgentArgs := &ai.AnalystAgentArgs{
```

Comment:
nit: maybe only create this when `req.Explore != ""` – mainly just to keep logs cleaner
```
var analystAgentArgs *ai.AnalystAgentArgs
if req.Explore != "" {
  analystAgentArgs = &ai.AnalystAgentArgs{
    ...
}